controllers/email.js

```diff
@@ -0,0 +1,50 @@
+const nodemailer = require('nodemailer')
+
+module.exports = {
+  sendemail: function(user) {
+    // Storing the sender email details
+    var transporter = nodemailer.createTransport({
+      service: 'gmail',
+      auth: {
+        user: 'havetocreateone@gmail.com',
+        pass: 'password',
+      },
+    })
+
+    //Setting up the handlebars with the nodemailer
+
+    // transporter.use(
+    //   'compile',
+    //   hbs({
+    //     viewEngine: {
+    //       extname: '.hbs',
+    //       defaultLayout: 'template',
+    //       partialsDir: '',
+    //     },
+    //     viewPath: '../views/',
+    //     extName: '.hbs',
+    //   }),
+    // )
+
+    // mailOptions contains the message to be sent to the registered user
+
+    var mailOptions = {
+      from: 'isha.venkatesh06@gmail.com',
+      to: 'sivasanjay7@gmail.com',
+      subject: 'Registration Success',
+      // text: `Welcome ${user.first_name}`,
+      html: '<h1>Explore the World! <h1> <br> ',
+      // template: 'index',
+    }
+
+    // sendMail to send the mail
+
+    transporter.sendMail(mailOptions, function(error, info) {
```

Comment:
Not immediately, but try to abstract this function and return a promise.
Like you should be able to write the below code across all functions.
```
send(mailOptions).then()
```